salt/utils/systemd.py

```diff
@@ -51,6 +52,27 @@ def booted(context=None):
     return ret
 
 
+def offline(context=None):
+    """Return True is systemd is in offline mode"""
```

Comment:
```suggestion
    """
    .. versionadded:: 3004
    
    Return True is systemd is in offline mode"""
```

Reply:
Done, but I added the versionadded after the description.